connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Herder.java

```diff
@@ -123,6 +123,10 @@
      */
     void putTaskConfigs(String connName, List<Map<String, String>> configs, Callback<Void> callback);
 
+    Collection<String> connectors();
```

Comment:
Please add JavaDoc to these methods, similar to the other methods surrounding this.